draft-welzl-ccwg-constrained-increase.md

```diff
@@ -111,19 +120,21 @@ cwnd_new = cwnd + SMSS*SMSS/cwnd
 cwnd = min(cwnd_new, 1+maxFS)
 ~~~
 
-maxFS is the largest FlightSize value since the last time cwnd was decreased. If cwnd has never been decreased, it is the maximum FlightSize value since the beginning of the data transfer.
+maxFS is the largest FlightSize value since the last time that cwnd was decreased.
+If cwnd has never been decreased, it is the maximum FlightSize value since the beginning of the data transfer.
 
 ## Discussion
 
-If the sender is limited for multiple RTTs, either by the application or by rwnd, continuously increasing cwnd can cause a mismatch between cwnd and the capacity the path provides. Such unlimited increase is therefore disallowed by the first rule.
-
-However, in most common congestion control mechanisms, in the absence of an indication of congestion, a cwnd that has been fully utilized during an RTT grants an increase during the immediately following RTT. Thus, such an increase is allowed by the second rule.
+If the sender rate is limited for multiple RTTs, either by the sending application or by the receivin applicatiom, continuously increasing the cwnd would cause a mismatch between the cwnd and the capacity the path provides (i.e., over-estimating the capacity).
```

Comment:
```suggestion
If the sender rate is limited for multiple RTTs, either by the sending application or by the receiving application, continuously increasing the cwnd would cause a mismatch between the cwnd and the capacity the path provides (i.e., over-estimating the capacity).
```